docs/user_manual/calculations.md

```diff
@@ -112,7 +114,16 @@ Factorizing the matrix of linear equation is the most computationally heavy task
 
 
 #### Linear
-It will be more accurate when most of the load/generation types are of constant impedance.
+
+This is an approximation method where we assume that all loads and generations are of constant impedance type regardless of their actual `LoadGenType`.
+By doing so, we obtain huge performance benefits as the computation required is equivalent to a single iteration of the iterative methods.
+It will be more accurate when most of the load/generation types are of constant impedance or the actual node voltages are close to 1 p.u.
```

Comment:
Mention here that if all the loads have `LoadGenType.const_impedance`. Linear method will be always used regardless of the actual chosen calculation method. Because the linear method can produce the accurate result.